metricbeat/tests/system/test_cmd.py

```diff
@@ -124,6 +124,21 @@ def test_modules_test(self):
         assert self.log_contains("cpu...OK")
         assert self.log_contains("memory...OK")
 
+    def test_modules_test_with_module_in_main_config(self):
+        self.render_config_template(reload=False, modules=[{
+            "name": "system",
```

Comment:
Maybe to add a `processors` field so as to be more close to the PR description?

Reply:
Processors are not needed to reproduce the issue. The thing is that before adding support for processors in light modules, Metricbeat metricsets didn't need a publisher pipeline during initialization, but they need it now, no matter if they use processors or not.

Reply:
👍